src/snovault/elasticsearch/__init__.py

```diff
@@ -49,7 +49,7 @@ def datastore(request):
     if request.__parent__ is not None:
         return request.__parent__.datastore
     datastore = 'database'
-    if request.params.get('frame') == 'edit':
+    if request.params.get('frame') == 'edit' or request.params.get('frame') == 'history':
```

Comment:
Not sure if it breaks anything to do `if request.params.get('frame') in ['edit', 'history']:`? Might be an issue if you specify multiple `&frame=` in the URL?

Reply:
Seems equivalent. What should happen if someone submits more than one frame currently?

Reply:
I think `params.get` just returns the last value, `params.getall` would return a list. https://docs.pylonsproject.org/projects/webob/en/stable/api/multidict.html 

Since this code runs on every request calling `request.params.get('frame')` once and seeing if a list contains the value seems like it would be slightly more efficient than calling `request.params.get('frame')` twice. On other hand might not make a significant difference for two values. Would probably have to measure to know for sure.